datadog_checks_base/datadog_checks/base/utils/http.py

```diff
@@ -89,6 +93,7 @@
     'tls_use_host_header': False,
     'tls_ignore_warning': False,
     'tls_private_key': None,
+    'tls_protocol': DEFAULT_PROTOCOL_VERSIONS,
```

Comment:
```suggestion
    'tls_protocols': DEFAULT_PROTOCOL_VERSIONS,
```
It's more than one

Reply:
I think the option should still start with `tls_*` like all other tls options, so i'll go for `tls_protocols_allowed`